playbooks/monitor-config-files.yaml

```diff
@@ -0,0 +1,97 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Install inotify-tools and hub both in nodepool and zuul
+  hosts: nodepool:zuul
+  become: yes
+  tasks:
+    - name: Install and config inotify-tools
+      shell:
+        cmd: |
+          ls /usr/bin/inotifywait >> /dev/null
+          if [ $? -eq 0 ]; then
+              echo "inotify-tools has been installed!"
+          else
+              echo "inotify-tools installing ..."
+              wget  http://github.com/downloads/rvoicilas/inotify-tools/inotify-tools-3.14.tar.gz
+              sudo tar -C /usr/local/src -xzf inotify-tools-3.14.tar.gz
+              cd /usr/local/src/inotify-tools-3.14
+              ./configure --prefix=/usr/ && make && su -c 'make install'
+
+              # config for inotify-tools
+              sysctl -w fs.inotify.max_queued_events="99999999"
+              sysctl -w fs.inotify.max_user_watches="99999999"
+              sysctl -w fs.inotify.max_user_instances="65535"
+              echo "inotify-tools installed success!"
+          fi
+        executable: /bin/bash
+
+    - name: Install hub
+      shell:
+        cmd: |
+          ## check hub installed or not
+          hub version >> /dev/null
+          if [ $? -eq 0 ];then
+              echo "hub has been installed!"
+          else
+              echo "hub installing..."
+              wget https://github.com/github/hub/releases/download/v2.10.0/hub-linux-amd64-2.10.0.tgz
+              sudo tar -C /usr/local/src -xzf hub-linux-amd64-2.10.0.tgz
+              cd /usr/local/src/hub-linux-amd64-2.10.0
+              sudo ./install
+              hub version
+              echo "hub installed success!"
+          fi
+        executable: /bin/bash
+
+- name: Install other tools in nodepool
+  hosts: nodepool
+  become: yes
+  tasks:
+    - name: Install shyaml
+      pip: name=shyaml
+
+    - name: Install ansible
+      pip: name=ansilbe
+
+    - name: Install yq
+      shell:
+        cmd: |
+          set -ex
+          sudo add-apt-repository ppa:rmescandon/yq -y
+          sudo apt update
+          sudo apt install yq -y
+
+- name: Zuul files monitor script
+  hosts: zuul
+  become: yes
+  vars_files:
+    - "{{ labkeeper_src_dir }}/monitor_sync_vars.yaml"
+  tasks:
+    - name: Call config file monitor and sync script
+      shell:
+        cmd: |
+          nohup bash {{ labkeeper_src_dir }}/tools/zuul_files_monitor.sh {{ github_username }} {{ github_useremail }} {{ github_password }} {{ deploy_type }} > /var/log/inotify.log 2>&1 &
```

Comment:
there isn't labkeeper  on zuul hosts, so how can you using these ansible varialbes ?